openeo_pg_parser_networkx/graph.py

```diff
@@ -32,31 +32,14 @@
 
 @dataclass
 class EvalEnv:
-    """
-    Object to pass which parameter references are available for each node throughout walking the graph.
```

Comment:
Why drop the doc string?

Reply:
Because it's not true anymore! Can add a short different one though!

Reply:
Yeah something would be good, if anyone else works on it just some basic info on what is expected would be appreciated!